server/db/models/announcement.model.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  Model,
+  Table,
+  Column,
+  HasMany,
+  Default,
+  AllowNull,
+  ForeignKey,
+  DataType,
+} from "sequelize-typescript";
+
+import { AnnouncementImage } from "./announcementimage.model";
+
+@Table({ tableName: "announcement" })
+export class Announcement extends Model {
+  @AllowNull(false)
+  @Column
+  title!: string;
+
+  @Column subtitle!: string;
+
+  // This was put because the input of the content is HTML form and varchar(255) is
+  // not enough to store all the info
+  @Column({
+    type: DataType.STRING(99999),
```

Comment:
This is too dangerous. Should at least add 'TODO' comment to investigate on restricting size